code/front/js/productFront.js

```diff
@@ -0,0 +1,148 @@
+/*
+** single product page
+*/
+
+//get _id of product from query parameters
+const queryString = window.location.search;
+console.log(queryString);
+let splitForId = queryString.slice(5);
+console.log(splitForId);
+
+const newUri = 'http://localhost:3000/api/products/' + splitForId;
+let product;
+
+/*
+** fetch
+*/
+
+//fetch the new URI from get request
+fetch(newUri) 
+    .then((response) => response.json())
+    .then((data) => {
+        console.log(data)
+        createIndividualProductView(data);
+        populateDropdown(data);
+        product = data;
+}); //gives back an individual object
+
+// dont need iterate over object to get keys / values -
+// already have in the console
+
+/*
+** create Product View
+*/
+
+
+//populate data, image ,description, price using keys/values
+function createIndividualProductView(data) {
+    //Dom access to existing ids + classes
+    let individualProductName = document.getElementById('title');
+    let individualProductPrice = document.getElementById('price');
+    let individualProductDescription = document.getElementById('description');
+    
+    let imgParent = document.getElementById('item__img');
+
+    // create element 
+    let individualProductImg = document.createElement('img');
+
+    //populate 
+    individualProductName.innerText = data.name;
+    individualProductPrice.innerText = data.price;
+    individualProductDescription.innerText = data.description;
+    individualProductImg.src = data.imageUrl;
+
+    //append
+    imgParent.appendChild(individualProductImg);
+
+    return createIndividualProductView;
```

Comment:
again,, you are returning a value and not using it ....., see line #23, if you do use it at the calling method, don't return it.

read about java functions, your evaluator will ask this, and you can fail the presentation if you don't know how to explain when to use a return value or not.